main.tf

```diff
@@ -9,23 +9,21 @@ terraform {
 provider "aws" {
   region = "us-west-2"
 }
-
-resource "aws_iam_role" "jay_project_role" {
-    name = "terraform_aws_lambda-role"
-    assume-role_policy = <<EOF
-  
-{
-    "Version": "2012-10-17",
-    "Statement".[
-        {
-            "Action": "sts:assumeRole,
-            "Principal": {
-                "Service": "lambda.amazonaws.com"
-            }
-            "Effect": "Allow",
-            "Sid": ""
-        },
-    ]
+data "aws_caller_identity" "current" {}
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+  iam_role   = join(":", ["arn:iam:", local.account_id, "role/labRole"]) 
```

Comment:
You did a mistake with the role nam

it starts with arn:aws:iam:

and ends with 

role/LabRole

so it has to be:
`iam_role   = join(":", ["arn:aws:iam:", local.account_id, "role/LabRole"]) `

Reply:
ok thanks